build/ci/evergreen.yml

```diff
@@ -150,6 +166,37 @@ functions:
           LC_GROUP_ID: ${automation_agent_mms_group_id}
           BASE_URL: ${mcli_ops_manager_url}
         args: ['-u', 'ubuntu', '-i', 'ssh_id', '-h', 'hosts.json']
+  "install ops manager 4.4":
```

Comment:
how do I know when I want to install 4.2 vs 4.4 inside a script? I think that is much easier to define a new evergreen operation to handle this instead using bash. (I personally prefer to use bash as less as possible)

Reply:
you don't need bash at all, look how we do `"e2e test"`, is a function and we pass different parameters depending if it's Atlas or cloud manager, same applies here, you can have a single `install ops manager` function that receives  `ARCHIVE` as a parameter. this goes along with your comment on replacing `"e2e test ops manager":` with the already existing `"e2e test"`, why does it fail? it probably misses a couple of settings that you need to pass to the call of  `"e2e test"`

Reply:
got it! Thanks. Regarding `e2e test`, it is passing more expansions and env var than `e2e test ops manager` so my guess is that the e2e tests are using those vars instead of the toml.

Reply:
`e2e test` just exports them, if they are not set when you call it then it should be ok, my bet is you are not passing the `XDG_CONFIG_HOME` which is the only difference I see, again you may want to debug that as I don't think we should need a new way to run the e2e tests

Reply:
it was failing because `e2e test` was setting `MCLI_OPS_MANAGER_URL`